### PR TITLE
reorganize virtwrap/cache NamespaceKey functions

### PR DESCRIPTION
**Pull request by mpolednik**
The functions had two issues: they had been placed in different packages and didn't really follow golang's naming convention. This PR solves both, moving the functions to virtwrap package.

The final destination should be the cache package, but that requires more refactoring to avoid  cyclic imports.

### CODE REVIEW

**Comment by mpolednik** [comment]
Maybe point for discussion: I'm thinking of moving the libvirt connection handling under cache while leaving the manager (KillVM, SyncVM, iface, helpers) stay in virtwrap. Opinions?

File: pkg/virt-handler/rest/console.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func (t *Console) Console(request *restful.Request, response *restful.Response)
 	namespace := request.PathParameter("namespace")
 	vm := v1.NewVMReferenceFromNameWithNS(namespace, vmName)
 	log := logging.DefaultLogger().Object(vm)
-	domain, err := t.connection.LookupDomainByName(virtwrap.VMNamespaceKeyFunc(vm))
+	domain, err := t.connection.LookupDomainByName(virtwrap.VmNamespaceKeyFunc(vm))
```
Comment (rmohr) [inline pkg/virt-handler/rest/console.go]: Go uses VMNamespaceKeyFunc, see https://github.com/golang/go/wiki/CodeReviewComments#initialisms

**Comment by rmohr** [comment]
> Maybe point for discussion: I'm thinking of moving the libvirt connection handling under cache while leaving the manager (KillVM, SyncVM, iface, helpers) stay in virtwrap. Opinions?

Maybe moving it to a cli package alongside cache?

File: pkg/virt-handler/rest/console.go (hunk 1 above)
Comment (mpolednik) [inline pkg/virt-handler/rest/console.go]: Cool, couldn't find this source. Will drop the renaming for sure!

**Comment by mpolednik** [comment]
Took a bit longer approach to the move. The PR now contains commit that

1. split libvirt client code from manager.go to virtwrap/cli/libvirt.go,
2. moves errors under virtwrap/errors package,
3. moves VMNamespace funcs to the cache package.

**Comment by rmohr** [comment]
ok to test

**Pull request merged**